pkg/controller/config/v1alpha1/doc.go

```diff
@@ -0,0 +1,6 @@
+// +k8s:deepcopy-gen=package
```

Comment:
New files should include the SPDX license header.

Reply:
They will now with new codegen.